boefjes/boefjes/plugins/kat_dns/main.py

```diff
@@ -17,44 +18,50 @@ class ZoneNotFoundException(Exception):
     pass
 
 
+def get_record_types() -> List[str]:
+    dns_record_types = ["A", "AAAA", "CAA", "CERT", "RP", "SRV", "TXT", "MX", "NS", "CNAME", "DNAME"]
+    requested_record_types = getenv("RECORD_TYPES", False)
+    if not requested_record_types:
+        return dns_record_types
+    requested_record_types = re.sub("[^A-Za-z,]", requested_record_types.upper()).split(',')
+    return list(set(requested_record_types).intersection(set(dns_record_types))
+
+                
 def run(boefje_meta: BoefjeMeta) -> List[Tuple[set, Union[bytes, str]]]:
     hostname = boefje_meta.arguments["input"]["name"]
 
     requested_dns_name = dns.name.from_text(hostname)
-    zone_soa_record = get_parent_zone_soa(requested_dns_name)
+    resolver = dns.resolver.Resolver()
+    nameserver = getenv("REMOTE_NS", str(settings.remote_ns))
```

Comment:
`settings` doesn't reference anything?

Reply:
See line 11

Reply:
Ah my bad, missed this one. I noticed that `kat_dns`, `kat_rdns`, `kat_dnssec` and `kat_manyportsopen` have a reference to the settings. However, I assumed that these references were removed after we implemented KATalogus settings. The latter would also make more sense, in the context of isolated entities/ containers that eventually get their input from the environment instead. Let's discuss this soon if you're on the same line

Reply:
I agree, there should be a few very clear input options for each defined Boefje Being:
* Input-OOI
* Env-vars
* parameters
* Config

Those should be configurable in a mix of katalogus settings, local env-vars and whatever the scheduler sends as the input-OOI.
There might be an overlap in the parameters, config and env to accomodate for global settings etc.